Documentation/CUSTOMIZE.md

```diff
@@ -134,9 +134,28 @@ When installing or upgrading an active-standby, geo-redundant cluster, all 6 VSD
 
 `vstats.yml` contains the definition of the VSTATs (VSD Statistics) to be operated on in this deployment. This file should be present in your deployment only if you are specifying VSTATs. If not provided, no VSTATs will be operated on. This file is of yaml list type. If it contains exactly 3 VSTAT definitions, a cluster installation or upgrade will be executed. Any other number of VSTAT definitions will result in 1 or more stand-alone VSTATs being installed or upgraded.
 
+## VSD RTT Performance Testing
+
+You can use MetroAE to verify that your VSD setup has sufficient RTT performance. By default, the RTT performance test will run at the beginning of the VSD deploy step, prior to installing the VSD software. The parameters that you can use to control the operation of the test are available in 'common.yml':
+
+* `vsd_run_cluster_rtt_test` When true, run RTT tests between VSDs in a cluster or standby/active cluster, else skip the test
+* `vsd_ignore_errors_rtt_test` When true, continue metroae execution upon error and do not validate the RTT between VSDs in a cluster is less than max RTT, else stop metorae execution upon error
```

Comment:
Use "MetroAE" instead of "metroae" when not specifying command syntax. There are a couple of instances in this file where "metroae" is used.